pkgs/development/python-modules/torchaudio/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, pkg-config
+, ninja
+, pybind11
+, torch
+}:
+
+buildPythonPackage rec {
+  pname = "torchaudio";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "pytorch";
+    repo = "audio";
+    rev = "v${version}";
+    hash = "sha256-qrDWFY+6eVV9prUzUzb5yzyFYtEvaSyEW0zeKqAg2Vk=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'print(" --- Initializing submodules")' "return" \
+      --replace "_fetch_archives(_parse_sources())" "pass"
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    ninja
+  ];
+  buildInputs = [
+    pybind11
+  ];
+  propagatedBuildInputs = [
+    torch
+  ];
+
+  BUILD_SOX=0;
+  BUILD_KALDI=0;
+  BUILD_RNNT=0;
+  BUILD_CTC_DECODER=0;
+
+  dontUseCmakeConfigure = true;
+
+  doCheck = false; # requires sox backend
```

Comment:
I think it is better to set these options as arguments of this file.
torchaudio can support CUDA. 
Could you add the option of CUDA?

Reply:
Currently, none of these can be enabled, as they cause build issues that need to be resolved.

The CUDA support is needed, I agree.